docker/test_env/geth_test.go

```diff
@@ -0,0 +1,17 @@
+package test_env
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/chainlink-testing-framework/docker"
+)
+
+func TestGeth(t *testing.T) {
+	network, err := docker.CreateNetwork()
+	require.NoError(t, err)
+	g := NewGeth([]string{network.Name})
+	_, _, err = g.StartContainer()
```

Comment:
It's worth adding the creation of a blockchain.EVMClient here

Reply:
Good call, added